services/src/test-library/src/main/java/org/openkilda/testing/config/DefaultServiceConfig.java

```diff
@@ -66,8 +67,10 @@ public RestTemplate traffExamRestTemplate() {
     }
 
     @Bean(name = "lockKeeperRestTemplate")
-    public RestTemplate lockKeeperRestTemplate(@Value("${lockkeeper.endpoint}") String endpoint) {
-        return buildLoggingRestTemplate(endpoint);
+    public RestTemplate lockKeeperRestTemplate(
+            @Value("${lab-api.endpoint}") String baseEndpoint,
+            LabService labService) {
+        return buildLoggingRestTemplate(baseEndpoint + "/api/" + labService.getLab().getLabId() + "/lock-keeper");
```

Comment:
I don't understand how this will work for hardware environment at this point. If I execute a hardware profile it will create a virtual lab for me, won't it?